analysis/stripe/analysis.sql

```diff
@@ -0,0 +1,87 @@
+with
+
+invoices as (
+  select *
+  from {{env.schema}}.stripe_invoices_transformed
+),
+
+plan_changes as (
+  select
+    invoices.customer as customer,
+    date_trunc('month', invoices.date) as month,
+    invoices.total as now,
+    prior_month_invoices.total as before,
+    invoices.total - prior_month_invoices.total as change
+  from
+    invoices
+    left outer join
+      invoices prior_month_invoices
+    on
+      date_trunc('month', add_months(invoices.date, -1)) = date_trunc('month', prior_month_invoices.date)
+      and invoices.customer = prior_month_invoices.customer
+  where
+    invoices.forgiven is not true
+    and invoices.paid is true
+),
+
+news as (
+  select
+    date_trunc('month', date) as month,
+    sum(total) as total
+  from invoices
+  where first_payment = 1
+  group by 1
+),
+
+churns AS (
+  select
+    date_trunc('month', invoices.date) as month,
```

Comment:
@jthandy Is the churn being recorded for the month of the last payment? Or should it be recorded the following month?